qa/L0_infer_zero/test.sh

```diff
@@ -55,6 +55,10 @@ rm -fr models && mkdir models
 cp -r /data/inferenceserver/${REPO_VERSION}/qa_identity_model_repository/* models/. && \
     cp -r /data/inferenceserver/${REPO_VERSION}/qa_ensemble_model_repository/qa_identity_model_repository/* models/.
 
+# Remove version-compatible TensorRT models, as they require version-compatibility
+# mode to be turned on when starting the server.
+rm -r models/plan_compatible*
```

Comment:
`-f` to avoid failure in case it's not there for some reason.
```suggestion
rm -rf models/plan_compatible*
```